assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11815,6 +11844,20 @@ protected int sizeResources(SessionState state) {
                     returnResources.addAll(assignmentService.getAssignmentsForContext((String) state.getAttribute(STATE_CONTEXT_STRING)));
                 }
 
+                //Filter assignments by group
```

Comment:
eventually sizeResources will be replaced with a more efficient method in the service!

Reply:
Yeah, makes sense